wolfcrypt/src/ecc.c

```diff
@@ -5795,12 +5795,19 @@ int wc_ecc_make_key(WC_RNG* rng, int keysize, ecc_key* key)
 WOLFSSL_ABI
 int wc_ecc_init_ex(ecc_key* key, void* heap, int devId)
 {
-    int ret = 0;
+    int ret      = 0;
+    int isPkcs11 = 0;
```

Comment:
Please wrap all `isPkcs11` in `HAVE_PKCS11`. I don't want to grow code size or have to have an "unused"